src/controllers/UsersController.js

```diff
@@ -344,7 +346,56 @@ class UsersController {
    * @param {object} req
    * @param {object} res
    * @memberof UsersController
-   * @returns successful pasword reset
+   * @returns successful profile update
+   */
+  static async updateProfile(req, res) {
```

Comment:
The function name should compliment the url in the description. It's creating some confusion.